filemanager.md

```
@@ -0,0 +1,28 @@
+```
+
+### Directory Opus
+docs: https://www.gpsoft.com.au/help/opus11/index.html#!Documents/Go1.htm
+```
+Path : (installed path)dopusrt.exe
+Arg For Folder : /cmd Go "%d" NEW
+Arg For File : /cmd Go "%f" NEW
+```
+
+### Total Commander
+docs: https://www.ghisler.ch/wiki/index.php/Command_line_parameters
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)